### PR TITLE
Remove unnecessary field.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
retest vsi please

**Comment by CyrusNajmabadi** [comment]
test vsi please

File: src/EditorFeatures/Core/Extensibility/Completion/CompletionHelper.cs [hunk 1]
```diff
@@ -15,11 +15,8 @@ namespace Microsoft.CodeAnalysis.Editor
 {
     internal class CompletionHelper
     {
-        public CompletionService CompletionService { get; }
-
         protected CompletionHelper(CompletionService completionService)
         {
```
Comment (balajikris) [inline src/EditorFeatures/Core/Extensibility/Completion/CompletionHelper.cs]: unused ctor parameter?

**Comment by balajikris** [comment]
:+1:

**Comment by CyrusNajmabadi** [comment]
retest vsi please

**Pull request merged**